Makefile

```diff
@@ -257,3 +259,16 @@ $(ISO_BUILD_IMAGE): deploy/iso/minikube-iso/Dockerfile
 release-iso: minikube_iso checksum
 	gsutil cp out/minikube.iso gs://$(ISO_BUCKET)/minikube-$(ISO_VERSION).iso
 	gsutil cp out/minikube.iso.sha256 gs://$(ISO_BUCKET)/minikube-$(ISO_VERSION).iso.sha256
+
+out/docker-machine-driver-kvm: $(KVM_DRIVER_FILES)
```

Comment:
I think renaming might be a lot of work.  We'd need to add it to the supported driver list in minikube, so it would show up for all users.  I was planning to drop official support for the docker-machine-driver-kvm when we start bundling the in-tree one.  There really should be no functionality difference between the two.